### PR TITLE
Translate the switcher

### PR DESCRIPTION
**Pull request by tvlooy**
It might be a good idea to translate the language switcher so people don't have to know English to find their language in it.

### CODE REVIEW

**Comment by sgolemon** [comment]
+1 on doing this from me.  It didn't quite sit right that these should be anglified.

File: releases/8.0/common.php [hunk 1]
```diff
@@ -58,12 +58,12 @@ function common_header(string $description): void {
...
-        'pt_BR' => 'Brazilian Portuguese',
-        'de'    => 'German',
-        'es'    => 'Spanish',
-        'fr'    => 'French',
-        'ru'    => 'Russian',
-        'zh'    => 'Chinese Simplified',
+        'de'    => 'Deutsch',
+        'es'    => 'Español',
+        'fr'    => 'Français',
+        'pt_BR' => 'Português do Brasil',
```
Comment (sgolemon) [inline releases/8.0/common.php:64]: Not sure why you changed the order as well, but I'm not going to worry about that for now.  IMO we should probably asort() this whole thing.

**Comment by php-pulls** [comment]
**Comment on behalf of pollita at php.net:**

Merged as 290632d2

File: releases/8.0/common.php (hunk 1 above)
Comment (tvlooy) [reply releases/8.0/common.php:64]: ah yes, asort() would be better :)

**Pull request closed without merging**